Taskfile.yml

```diff
@@ -101,6 +117,13 @@ tasks:
     cmds:
       - docker compose {{ .DOCKER_COMPOSE_FILES }} run --rm cli sh -c "{{.CLI_ARGS}}"
 
+  dev:bnfcli:
+    summary: dev:cli for BNF site, a noop if BNF not enabled
+    status:
+      - "[[ -z \"{{ .DOCKER_COMPOSE_FILES_BNF}}\" ]]"
```

Comment:
I think it would be nice with and error to the end user if BNF is not enabled when using this cmd.

Reply:
And I can see now it is because you use it internally in the Taskfile. The thing is that we use "dev:cli" directly quite a lot in our team. So, as it is with these changes, if we call `task dev:bnfcli` nothing would happen and the developer would not know why, right?